tedana/tests/test_mapca.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tests for maPCA
+"""
+
+import numpy as np
+import nibabel as nib
+from tedana import decomposition
+
+
+def test_ent_rate_sp():
+    """
+    Check that ent_rate_sp runs correctly, i.e. returns a float
+    """
+    test_data = np.random.rand(200, 10, 10)
+    ent_rate = decomposition.ent_rate_sp(test_data, 1)
+    assert type(ent_rate) is np.float64
```

Comment:
```suggestion
    assert isinstance(ent_rate, float)
```

Reply:
Regarding the if statements, I believe we should always be in the last scenario (i.e. 3 dimensions where none is size 1). However, the original GIFT code had these other two cases that we decided to translate from MATLAB just in case. What I want to say is, we may want to remove those if statements that won't be used (our input will always be 3D) and could be the source of future issues.

So, if you all agree, I'll get rid of the first two if statements @javiergcas @rmarkello @emdupre @jbteves @tsalo 

Checking for the `ValueError`s sounds good.

Reply:
We can definitely remove the first two if-statements; might want to add a shape check at the top of the function, then !